### DISCUSSION TITLE
How to sort lines in a selection?

### DISCUSSION THREAD

**Discussion by Robert-M-Muench** [helix-editor/helix#1366] [Q&A] [answered]
See subject.

**Comment by Omnikar** [comment]
You can probably pipe the selection through a shell command using `|` in order to do that.

**Reply by hovsater** [reply]
This is what I currently use since its's actually faster and shorter to type than using the builtin `:sort` command. With `:sort` you have to make a selection, press `Alt-s` and then run `:sort`. Using a pipe it's as easy as `|sort`.

**Comment by sudormrfbin** [comment, marked as answer]
There's also a built-in `:sort` command being worked on in #1288.

**Reply by kirawi** [reply]
Merged in a4641a8613bcbe4ad01d28d3d2a6f4509fef96a9

**Reply by matoous** [reply]
So the current answer is: select the lines, press `Alt+s` to split the selection on new lines, and call `:sort` to sort the lines (`:rsort` for reverse).
